test/unit/io.js

```diff
@@ -735,6 +779,12 @@ describe('Input/output', function () {
     });
   });
 
+  it('Invalid WebP alpha quality throws error', function () {
```

Comment:
Invalid `lossless` and `nearLossless` attributes are ignored and are set to default boolean `false`. Similar to what is done for `jpegOvershootDeringing` and other such params